app/App_Resources/iOS/build.xcconfig

```diff
@@ -6,5 +6,7 @@
 ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 CODE_SIGN_ENTITLEMENTS = fieldkit/fieldkit.entitlements;
-IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 VALIDATE_WORKSPACE = YES;
+
+DEVELOPMENT_TEAM = U8DFGB86GQ
```

Comment:
Is this safe to commit? Probably shouldn't have the team?

Reply:
Sorry, I thought I removed the changes here. Fixed it with a commit.